spec.html

```diff
@@ -14109,7 +14109,7 @@ <h1>
           1. Let _succeeded_ be ! OrdinaryDefineOwnProperty(_A_, *"length"*, _newLenDesc_).
           1. If _succeeded_ is *false*, return *false*.
           1. For each own property key _P_ of _A_ that is an array index, whose numeric value is greater than or equal to _newLen_, in descending numeric index order, do
-            1. Let _deleteSucceeded_ be ! _A_.[[Delete]](_P_).
+            1. Let _deleteSucceeded_ be ! <emu-meta effects="user-code">_A_.[[Delete]]</emu-meta>(_P_).
```

Comment:
Because we know this is an Array exotic object, we know that this does not actually invoke user code. It's just the ordinary delete.

The first two uses of OrdinaryDefineOwnProperty in this method also do not invoke user code. I'm pretty sure they could be prefixed with `!`, so as with the above-mentioned unprefixed DefinePropertyOrThrow uses, #2547 will maybe fix them.